pkg/plugin/javascript.go

```diff
@@ -390,14 +390,16 @@ func (t *jsPlugin) PrintTab(ctx context.Context, object runtime.Object) (TabResp
 	if contents, ok := contents["contents"]; ok {
 		realComponent, err := javascript.ConvertToComponent("tab contents", contents)
 		if err != nil {
-			return TabResponse{}, fmt.Errorf("unable to extract component: %w", err)
+			return []TabResponse{}, fmt.Errorf("unable to extract component: %w", err)
 		}
 		realFlexLayout := *realComponent.(*component.FlexLayout)
 		cTab.Contents.Config = realFlexLayout.Config
 	}
 
-	return TabResponse{
-		Tab: cTab,
+	return []TabResponse{
+		{
+			Tab: cTab,
```

Comment:
So for now JS plugins can still generate a single tab? I'm guessing there will be updates here after corresponding changes are made pushed https://github.com/vmware-tanzu/plugin-library-for-octant?

Reply:
Good call out, we'll want to update https://github.com/vmware-tanzu/plugin-library-for-octant/blob/main/plugin/helpers.ts#L83 and https://github.com/vmware-tanzu/plugin-library-for-octant/blob/main/plugin/octant.d.ts#L62 after this lands.